src/main/java/com/accelad/docx4j/tags/TagUpdaterTable.java

```diff
@@ -60,12 +62,12 @@ public boolean update(Tag tag) {
     }
 
 
-    private TableRow createHeaders(String[] headers) { // NOSONAR
+    private TableRow createHeaders(TagValueTableRow headers) { // NOSONAR
```

Comment:
Mhh.. honestly I don't know why it was here.
SonarLint tells me it is OK without it.
I see no reasons to keep it  -> removal